de.tudarmstadt.ukp.wikipedia.util/src/main/java/de/tudarmstadt/ukp/wikipedia/util/templates/generator/simple/WikipediaTemplateInfoDumpWriter.java

```diff
@@ -227,41 +222,26 @@ private String generateRevisionSQLStatement(boolean tableExists,
 	 * @throws Exception
 	 */
 	public void writeSQL(boolean revTableExists, boolean pageTableExists,
-			GeneratorMode mode)
-		throws Exception
+			GeneratorMode mode) throws Exception
 	{
-		Writer writer = null;
-		try {
-			writer = new BufferedWriter(new OutputStreamWriter(new BufferedOutputStream(
-					new FileOutputStream(outputPath)), charset));
-			StringBuffer dataToDump = new StringBuffer();
+		try (Writer writer = new BufferedWriter(new OutputStreamWriter(
+				new BufferedOutputStream(new FileOutputStream(outputPath)), charset))){
+			StringBuilder dataToDump = new StringBuilder();
 
 			dataToDump.append(generateTemplateIdSQLStatement(this.tableExists));
 
 			if (mode.active_for_pages) {
 				dataToDump.append(generatePageSQLStatement(pageTableExists,
 						mode.templateNameToPageId));
 			}
-
 			if (mode.active_for_revisions) {
 				dataToDump.append(generateRevisionSQLStatement(revTableExists,
 						mode.templateNameToRevId));
 			}
-
 			writer.write(dataToDump.toString());
 		}
 		catch (IOException e) {
-			logger.error("Error writing SQL file: " + e.getMessage());
-		}
-		finally {
-			try {
-				if (writer != null) {
-					writer.close();
-				}
-			}
-			catch (IOException e) {
-				logger.error("Error closing stream: " + e.getMessage());
-			}
+			logger.error("Error writing SQL file: " + e.getMessage(), e);
```

Comment:
Should be using placeholders, not string concatenation.

Reply:
Fixed with next commit.